packages/engine/lib/execution/src/error.rs

```diff
@@ -31,6 +31,9 @@ pub enum Error {
     #[error("Arrow Error: {0}")]
     Arrow(#[from] arrow::error::ArrowError),
 
+    #[error("IO error: {0:?}")]
+    IO(#[from] std::io::Error),
```

Comment:
It will look gross but shouldn't this be `Io`?

Reply:
We could call it `StdIo` to get around it 😂

Reply:
Let's call it `MoonOfJupiter` 🙃